index.bs

```diff
@@ -780,92 +776,54 @@ This is the "code smell" mentioned above that suggests
-but the resulting behavior is different in a number of observable ways.
-
-As an inherited property,
-'text-decoration' would draw underlines (or other decorations)
-in the style and position of each piece of text.
-As a non-inherited property, it would draw underlines
-in the style and position of the element that specifies 'text-decoration'.
-These behaviors lead to different results
-when the element that has 'text-decoration' declared
-has descendants that are at a different vertical-alignment (e.g., superscripts),
-in a different color,
-or in a different font size.
-
-The property was designed as not inherited
-because the behavior of having underlines
-have a consistent color, thickness, and position
-was considered the better behavior.
-
-However, in the end, some of the characteristics of the other behavior
-(such as its interaction with 'visibility' or text selection)
-were also seen as desirable, and it ended up being specified in a somewhat hybrid way
-that does involve the "code smell"
-of walking up the ancestor chain looking for text decorations on ancestors.
-</div>
-
 <h3 id="css-computed-value">Choose the computed value type based on how the property should inherit</h3>
 
-Another important choice in the design of every CSS property
-is what its <a>computed value</a> is.
-This is an abstract description of the value that is inherited when the property inherits.
-It is also important for starting and animation of [=CSS Transitions=].
-This is most important for inherited properties, but it does matter for all properties.
-
-Note: Due to a somewhat unfortunate history,
-this is *not* the value that is returned by {{getComputedStyle()}},
-which is actually the <a>resolved value</a>,
-which can be the result of further computation starting from the computed value.
-
-The definition of a CSS property often has interactions with other CSS properties.
-It is often possible for these interactions to happen at the [=computed value=] stage
-or at a later stage such as the [=used value=].
-Since inheritance operates on the computed value,
-whether an interaction should affect the value that descendants inherit
-is a major factor in the choice of where that interaction should be handled.
-
-<a href="https://wiki.csswg.org/spec/property-dependencies">The dependencies
-that do affect computed values</a>
-are required to not be circular.
-
-Making decisions about whether dependencies should affect computed values
-often involves tradeoffs of complexity.
-Sometimes it is less complex
-for the behavior to happen as part of the computation of the computed value;
-sometimes it is less complex
-for the behavior to happen in whatever handles the computed value.
-(This complexity might be in the specification, in implementations, or both.
-Complexity in implementations might affect how fast things run for end users.)
-
-For more information about computed values, see
-[Computed Values Patterns](https://wiki.csswg.org/spec/computed-values).
```

Comment:
I'd like to try to get many of these points back in the document... although I admit they weren't very clear and lacked examples.  I'm hoping to try to write something soon...